src/main/java/com/linecorp/armeria/server/http/tomcat/TomcatService.java

```diff
@@ -73,6 +76,10 @@ public static TomcatService forFileSystem(Path docBase) {
         return TomcatServiceBuilder.forFileSystem(docBase).build();
     }
 
+    public static HttpService forConnector(String hostname, Connector connector) {
```

Comment:
Because connector is only a part of Tomcat? :-)
```
        //
        // server <------ services <------ engines <------ realm
        //                                         <------ hosts <------ contexts
        //                         <------ connectors
        //                         <------ executors
        //
```